recipe/shopware6.php

```diff
@@ -3,9 +3,12 @@
 
 set('repository', 'git@github.com:shopware/production.git');
 
+set('release_name', function () {
+    return date('YmdHis');
```

Comment:
Maybe this more user decided?

Reply:
Might be, but I've never seen somebody *not* change it to the timestamp.

Reply:
That's right, @peterjaap, but I wouldn't set this on an official recipe, as all the others don't change this also… BTW:  I've set it to this:
```php
set('release_name', static function (): string {
    return run('date +"%Y-%m-%d__%H-%M-%S"');
});
```

The benefit of that is that the release name is based on the server time and not on the local time.